Library/Formula/docker-machine.rb

```diff
@@ -22,9 +22,13 @@ def install
     mkdir_p buildpath/"src/github.com/docker/"
     ln_sf buildpath, buildpath/"src/github.com/docker/machine"
 
-    system "make", "build"
-    bin.install Dir["bin/*"]
+    Dir.chdir(buildpath/"src/github.com/docker/machine/") do
+      # Hack to force ruby chdir into the symlinked path
+      ENV["PWD"] = buildpath/"src/github.com/docker/machine/"
+      system "make", "build"
+    end
```

Comment:
You can delete the symlink if necessary. We need to not do a `go get` though; we can't have it downloading stuff in `install`. It should be submitted to `docker-machine` upstream because every other Go package works fine without this hack.